pkg/gadgets/traceloop/tracer/bpf/traceloop.bpf.c

```diff
@@ -230,8 +230,11 @@ int ig_traceloop_e(struct bpf_raw_tracepoint_args *ctx)
 	}
 
 	bpf_debug_printk("Perf event output: sc.id: %d; sc.comm: %s; sizeof(sc): %d\n", sc.id, sc.comm, sizeof(sc));
-	bpf_perf_event_output(ctx, perf_buffer, BPF_F_CURRENT_CPU, &sc,
+	ret = bpf_perf_event_output(ctx, perf_buffer, BPF_F_CURRENT_CPU, &sc,
 			      sizeof(sc));
+	if (ret != 0) {
+		bpf_error_printk("Problem outputting perf event: %d", ret);
+	}
```

Comment:
Only half of the bpf_perf_event_output in this file are checked in this way. Is it intended?
Other gadgets don't check the result of bpf_perf_event_output. Did you have a problem with it?